src/modules/navigator/mission_base.cpp

```diff
@@ -564,8 +564,6 @@ void MissionBase::handleLanding(WorkItemType &new_work_item_type, mission_item_s
 			_mission_item.autocontinue = true;
 			_mission_item.time_inside = 0.0f;
 			_mission_item.vtol_back_transition = true;
-
-			_navigator->reset_position_setpoint(pos_sp_triplet->previous);
```

Comment:
is it possible that a vtol land is triggered without a valid previous waypoint? what will happen then in the FW pos ctrl?

Reply:
@RomanBapst what about this question?

Reply:
@tstastny Please have a look again. There is now a dedicated method for controlling backtransitions. If the previous waypoint is not valid it will default to waypoint following.

Reply:
@tstastny Should not be an issue anymore, if now the previous waypoint  is invalid, the controller will use the position at the start of the transition and the land point for line following.